location : No such module 'location' in macos

While I am trying to run the app in macos, not able to build the desktop app.

below error :

```
Launching lib/main.dart on macOS in debug mode...
Upgrading project.pbxproj
Warning: CocoaPods recommended version 1.10.0 or greater not installed.
Pods handling may fail on some projects involving plugins.
To upgrade see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

Running pod install...
Building macOS application...
/Users/pradeepkumar/Downloads/flutterlocation-master/location/example/macos/Flutter/GeneratedPluginRegistrant.swift:8:8: error: no such module 'location'
import location
       ^
/Users/pradeepkumar/Downloads/flutterlocation-master/location/example/macos/Flutter/GeneratedPluginRegistrant.swift:8:8: error: no such module 'location'
import location
       ^
/Users/pradeepkumar/Downloads/flutterlocation-master/location/example/macos/Flutter/GeneratedPluginRegistrant.swift:8:8: error: no such module 'location'
import location
       ^
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description
** BUILD FAILED **

Exception: Build process failed
```



`pradeepkumar@Pradeep-Kumar example % flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.26.0-13.0.pre.119, on Mac OS X 10.15.7 19H15 darwin-x64, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
[!] Xcode - develop for iOS and macOS
    ! Xcode 11.6.0 out of date (12.0.1 is recommended).
      Download the latest version or update via the Mac App Store.
    ! CocoaPods 1.9.3 out of date (1.10.0 is recommended).
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To upgrade see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.5)
[✓] Connected device (2 available)

! Doctor found issues in 1 category.
`

--- a/packages/location/macos/Classes/LocationPlugin.m
+++ b/packages/location/macos/Classes/LocationPlugin.m
@@ -1,1 +1,1 @@
-darwin/Classes/LocationPlugin.m
+../../darwin/Classes/LocationPlugin.m

